gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/util/OutboundMessageUtil.java

```diff
@@ -0,0 +1,39 @@
+package uk.nhs.adaptors.pss.translator.util;
+
+import org.hl7.v3.RCMRIN030000UK06Message;
+import org.springframework.stereotype.Component;
+
+@Component
+public class OutboundMessageUtil {
```

Comment:
i think this is all in xml parse util? could be the other type of payload though. I think this makes more sense as a class and title to me though.